_posts/2022-04-19-debt-reduction-in-the-product-roadmap.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "How to include refactoring into Product development roadmap? Reducing technical debt inspired by real options identification"
+author: [olga.dudzik]
+tags: [tech]
+---
+
+## Nowadays, technical debt can be considered a bread and butter of most IT-powered enterprises around the world.
```

Comment:
Wydaje mi się, że to mogłoby być zwykłe zdanie, bo jako śródtytuł będzie trochę długie